TaskManager/src/Kreta/TaskManager/Infrastructure/Persistence/Doctrine/ORM/Organization/DoctrineORMOrganizationRepository.php

```diff
@@ -42,11 +42,14 @@ public function query($specification)
     public function persist(Organization $organization)
     {
         $this->getEntityManager()->persist($organization);
+        $this->getEntityManager()->flush();
     }
 
+
     public function remove(Organization $organization)
     {
         $this->getEntityManager()->remove($organization);
+        $this->getEntityManager()->flush();
```

Comment:
Do you remember our conversation about remove items from collections in Doctrine?We decided to put this flush to force commit in bd...We spent a lot of time trying to fix this and this was the solution, if you want try to fix